vaadin-grid.html

```diff
@@ -6,6 +6,19 @@
 
 <link rel='import' href='../polymer/polymer.html'>
 <script src="vaadin-grid.min.js"></script>
+<script>
+if (!window.vaadin) {
```

Comment:
How is it a problem? If the namespace already exists then sdm.js simply isn't imported (as it shouldn't). We don't run sdm with Vaadin Framework during Core Elements development.